src/System.Runtime/tests/System/HashCodeTests.cs

```diff
@@ -0,0 +1,192 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+public static class HashCodeTests
+{
+
+#   if SYSTEM_HASHCODE_TESTVECTORS
+    [Theory]
+    [InlineData(0x02cc5d05U)]
```

Comment:
Where did you find these test vectors? How can verify them? Are there others we should include?

Reply:
Check the PR blurb for more info. I used [this site](https://asecuritysite.com/encryption/xxHash) with strings copied (increasing lengths of 4) from "abcd0123efgh4567ijkl8901mnop2345qrst6789uvwx0123yzab". [My original polyfill test](https://github.com/k2workflow/Clay/blob/master/src/SourceCode.Clay.Tests/HashCodeTests.cs#L18) actually self-documents it, but `Encoding` is not available at this callsite - so I turned them all into uint literals.

Reply:
Ah, I see now. Please add a comment to the code so that we can reproduce this in the future.